<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="initial_x" default="0" />
    <arg name="initial_y" default="0" />
    <arg name="des_x" default="-4" />
    <arg name="des_y" default="0" />
    <param name="initial_x" value="$(arg initial_x)" />
    <param name="initial_y" value="$(arg initial_y)" />
    <param name="way_x" value="$(arg des_x)" />
    <param name="way_y" value="$(arg des_y)" />
    <node pkg="indoor_nav" type="follow_wall.py" name="follow_wall" />
    <node pkg="indoor_nav" type="nav_to_waypoint.py" name="nav_to_waypoint" />
    <node pkg="indoor_nav" type="state_machine.py" name="state_machine" output="screen" />
</launch>